code/src/main/webapp/JavaScript/script.js

```diff
@@ -119,25 +139,32 @@ var startApp = function() {
   gapi.load('auth2', function() {
     // Retrieve the singleton for the GoogleAuth library and set up the client.
     auth2 = gapi.auth2.init({
+      apiKey: 'AIzaSyBMrfBNGcVEtoRsoduXvYSjd9piD36W7Qg',
```

Comment:
I'd check to make sure it's safe to check this into github + display it to the user (as it would be here if we served it to the user's browser).

Reply:
this code is from a different branch but I will look into it